fuzzing/build.gradle.kts

```diff
@@ -8,7 +8,17 @@ repositories {
 
 dependencies {
     implementation(project(":sigstore-java"))
+
+    implementation(platform("io.grpc:grpc-bom:1.46.0"))
```

Comment:
same here, limiting the dependencies to what is used.

Reply:
Thanks for the suggestion. Have removed unused dependencies.